scripts/build.sh

```diff
@@ -49,52 +49,6 @@ else
 fi
 
 #
-# run `npm install` to prepare the UI if it's changed since the last time we ran it
-#
-newest_commit=$(git rev-parse HEAD)
-last_commit_file="../.last_npm_installed_ui_commit"
```

Comment:
Are we sure we don't need this anymore? This file is referred here https://github.com/contiv/auth_proxy/blob/master/scripts/build.sh#L55 

I guess without `last_npm_installed_ui_commit ` npm will be installed every time we build.